CustomDropDown/View/CustomDropDown.swift

```diff
@@ -12,6 +12,18 @@ class CustomDropDown<T>: UIView {
     
     // MARK: - Variables
     
+    public var cornerRadius: CGFloat = 0.0 {
+        didSet{
+            containerView.layer.cornerRadius = cornerRadius
+            containerView.layer.masksToBounds = true
+        }
+    }
+    
+    // Making `masksToBounds = true` to have round corners prevent the shadow to appear. So, it was necessary to
+    // wrap the tableview in a containerView. This way, the shadow can be applied to the view while the
+    // round corners are applied to the containerView.
+    private let containerView = UIView()
```

Comment:
Do we need this containerView? Did you try on the normal tableview? Just wanted to confirm.

Reply:
You're right! There is no need for the additional container view. Applying the cornerRadius to the tableview worked fine.